### PR TITLE
data:Govamarket

### PR DESCRIPTION
**Pull request by Govamarket**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4580"></a>

### CODE REVIEW

File: data/Govamarket.jason [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "Clinton Chidera",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend developer",
+  "avatar": https://github.com/Govamarket.png"
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
```
Comment (priyankarpal) [inline data/Govamarket.jason:11]: You added the wrong username. Please add your username.
``` suggestion
      "url": "https://twitter.com/Your-user-name",
```

File: data/Govamarket.jason [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Clinton Chidera",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend developer",
+  "avatar": https://github.com/Govamarket.png"
```
Comment (priyankarpal) [inline data/Govamarket.jason:6]: Currently, you don't need to add an avatar. It will automatically fetch from your GitHub.
``` suggestion
 
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Please fix this spelling.
`data/Govamarket.jason`
to 
`data/Govamarket.json` 
Screenshot from 2023-02-07 17-37-48

**Comment by Govamarket** [comment]
These level is real.

File: data/Govamarket.jason [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "Clinton Chidera",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend developer",
+ 
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Your-user-name",
```
Comment (priyankarpal) [inline data/Govamarket.jason:11]: @Govamarket you need to add your Twitter account's username.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Please change `data/Govamarket.jason` to `data/Govamarket.json` otherwise, it will not work.
You can check this official doc. It will be easier for you.https://linkfree.eddiehub.io/docs/quickstart

File: data/Govamarket.jason (hunk 3 above)
Comment (Govamarket) [reply data/Govamarket.jason:11]: @Govamarket

Comment (Govamarket) [reply data/Govamarket.jason:11]: @Govamarket is my  Twitter account username.

**Review by Govamarket — COMMENTED** [review]
My Twitter account username is not complete.

**Comment by priyankarpal** [comment]
https://github.com/EddieHubCommunity/LinkFree/pull/4580#pullrequestreview-1287062154 

Please solve this issue. You need to update it.  @Govamarket it's mandatory. If you don't do it your profile wouldn't visible. I already mentioned it. Don't forget to check it. if you need help you can ask.

**Review by iamthecloverly — DISMISSED** [review]

**Review by Govamarket — COMMENTED** [review]
The json has been corrected

**Comment by Govamarket** [comment]
The json has been corrected.

**Review by priyankarpal — APPROVED** [review]
Now it's looking great. 🎉

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Review by Govamarket — COMMENTED** [review]
Name changed to Govamarket and merge took place.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**